.github/workflows/build-and-test.yml

```
@@ -70,6 +70,10 @@ jobs:
         git config --local user.email "action@github.com"
         git config --local user.name "GitHub Action"
         
+        echo "Switching to actual branch..."
+        git checkout ${{ github.head_ref }}
+        git pull origin ${{ github.head_ref }}
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)